engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/NodeDependency.scala

```diff
@@ -16,15 +16,18 @@ object Parameter {
 
   def apply[T: ClassTag](name: String): Parameter = Parameter(name, Typed[T], implicitly[ClassTag[T]].runtimeClass)
 
-  def apply(name: String, typ: TypingResult, runtimeClass: Class[_]): Parameter =
-    Parameter(name, typ, runtimeClass, editor = None, validators = List(MandatoryValueValidator), // we want to have mandatory parameters by default because it can protect us from NPE in some cases
+  def apply(name: String, typ: TypingResult, runtimeClass: Class[_], validators: List[ParameterValidator] = List.empty): Parameter =
+    Parameter(name, typ, runtimeClass, editor = None, validators = List(MandatoryValueValidator) ++ validators, // we want to have mandatory parameters by default because it can protect us from NPE in some cases
```

Comment:
Let's add new method for this, without default MandatoryValueValidator, now it can be confusing...

Reply:
MandatoryValueValidator should be as default.. NotBlank has sense after field is set..

Reply:
I think that @mproch rather have an idea to add `apply` method with `validators: List[ParameterValidator]` and make sure that you will pass there correct list of validators. @mproch am I right? I don't think that adding `notBlank` to `Parameter` is a good idea. We will have now additional method for all combinations of validators? Optional/Mandatory are most important cases and it is why we have special methods for them...

Reply:
I've changed it.. I've create object for NotBlankParam with method where I put array of validators.